app/controllers/api/v1/messages_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class MessagesController < ApplicationController
+      def index
+        @messages = Message.all
```

Comment:
- [ ] Hello, @Lancelot-SO GOOD JOB attempting to address the issue raised by the previous reviewer. However, your API continues to behave in the same manner, returning all greetings rather than just one at a time. Please make sure to correct that. 
#### Returning all the greeting :point_down: 
![Screenshot from 2023-06-06 13-50-10](https://github.com/Lancelot-SO/hello-rails-react/assets/89228743/86ed32f5-55e8-4248-b0ff-3005795b7726)



So, to help you, here's (below) :point_down:  one way to only return a random greeting from the database every time the page loads. By appending '.sample' to the end of your request. :ok_hand: 😊
```
@messages = Message.all.sample
```

```suggestion
  @messages = Message.all.sample
```